java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/Server.java

```diff
@@ -948,8 +949,14 @@ private void collectProjectCandidates(FileObject fo, List<FileObject> candidates
 
         private void initializeOptions() {
             getWorkspaceProjects().thenAccept(projects -> {
+                ConfigurationItem item = new ConfigurationItem();
+                item.setSection(NETBEANS_JAVA_HINTS);
```

Comment:
Should the section here be `client.getNbCodeCapabilities().getConfigurationPrefix() + NETBEANS_JAVA_HINTS`? What is exactly the setting that the user will see? `<prefix>.hints.preferences`?

Reply:
Yes `<prefix>.hints.preferences` is the configuration that the user will see.